config/routes.rb

```diff
@@ -206,9 +206,10 @@
     match '/manage/phone/:id' => 'users/edit_phone#update', via: %i[patch put]
     delete '/manage/phone/:id' => 'users/edit_phone#destroy'
     get '/manage/personal_key' => 'users/personal_keys#show', as: :manage_personal_key
-    post '/account/personal_key' => 'users/personal_keys#create', as: :create_new_personal_key
     post '/manage/personal_key' => 'users/personal_keys#update'
 
+    post '/account/personal_key' => 'accounts/personal_keys#create', as: :create_new_personal_key
```

Comment:
the fact that there were two separate prefixes `/manage` and `/account` was a hint to me that some things could be separated